docs/docs/integrations/providers/unstructured.mdx

```diff
@@ -8,10 +8,15 @@ ecosystem within LangChain.
 
 ## Installation and Setup
 
-If you are using a loader that runs locally, use the following steps to get `unstructured` and
-its dependencies running locally.
+If you are using a loader that runs locally, use the following steps to get `unstructured` and its
+dependencies running.
 
-- Install the Python SDK with `pip install unstructured`.
+- For the smallest installation footprint and to take advantage of features not available in the
+  open-source `unstructured package`, install the Python SDK with `pip install unstructured-client`.
```

Comment:
Should the "install `unstructured-client`" part go below in the section you removed about `UnstructuredAPIFileIOLoader`? Don't see the whole file in the diff, but generally:
- For `UnstructuredFileLoader`, tell people to install `unstructured`
- For `UnstructuredAPIFileLoader`, tell people to install `unstructured-client`

Reply:
Now that the API Loaders don't depend on `unstructured`, I wanted to add an install section for them here. The section below this wasn't completely accurate and seemed redundant once I added this part up here.
You can view the full diff/file [here](https://github.com/langchain-ai/langchain/pull/22779/commits/a172c1fe8aadbf9d1b3e5e6ca96f02dfe8841f2c)

I will update this to specifically mention the loaders, though